lib/active_merchant/billing/gateways/openpay.rb

```diff
@@ -24,6 +31,16 @@ def initialize(options = {})
         super
       end
 
+      def gateway_url(options = {})
+        country = options[:merchant_country] || @options[:merchant_country]
+
+        if country == 'MX'
+          test? ? mx_test_url : mx_live_url
+        else
+          test? ? co_test_url : co_live_url
```

Comment:
What I was suggesting in my request was to change this line to be `test? ? test_url : live_url`. They're assigned in line 11 & 12 but not used again. The code looks good and this is not a deal breaker so I'm going to approve your PR.